supported modules on AA (#2247)
RedisLabs/redislabs-docs: content/rs/databases/active-active/planning.md

## Redis modules {#redis-modules}

Several Redis modules are compatible with Active-Active databases. Find the list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
{{< note >}}

> Starting with v6.2.18, you can now index, query, and perform full-text searches of nested JSON documents in Active-Activate databases, by combining RedisJSON and RediSearch.

Reviewer: Noticed a typo.
```suggestion
Starting with v6.2.18, you can index, query, and perform full-text searches of nested JSON documents in Active-Active databases by combining RedisJSON and RediSearch.
```

> Starting v6.2.18 the capability of indexing, query and full-text search of nested JSON documents are now also GA on Active-Active (achieved by the combination of RedisJSON with RediSearch).

Reviewer 2: ```suggestion
Starting v6.2.18 the capability of index, query, and full-text search of nested JSON documents, are now also available for Active-Active databases by using a combination of RedisJSON with RediSearch.
```

> Starting v6.2.18 the capability of index, query, and full-text search of nested JSON documents, are now also available for Active-Active databases by using a combination of RedisJSON with RediSearch.

Reviewer 3: ```suggestion
Starting with v6.2.18, you can now index, query, and perform full-text searches of nested JSON documents in Active-Activate databases, by combining RedisJSON and RediSearch.
```
